gradescopeCronJob/gradescope_to_spreadsheet.py

```diff
@@ -82,7 +85,7 @@ def writeToSheet(sheet_api_instance, assignment_scores, assignment_name = ASSIGN
             sheet_id = response['replies'][0]['addSheet']['properties']['sheetId']
         else:
             sheet_id = sub_sheet_titles_to_ids[assignment_name]
-        time.sleep(5)
+        time.sleep(SLEEP_TIME)
         update_sheet_with_csv(assignment_scores, sheet_api_instance, sheet_id)
         print("Successfully updated spreadsheet with new score data")
```

Comment:
Can you change this print statement to be more specific? For example, what assignment just got updated?

Reply:
This has been resolved by Manan's code. Please fetch the latest changes from main.

Reply:
Great! Thank you, @mananb77 ! Will do.